code/game/objects/structures/mirror.dm

```diff
@@ -221,7 +221,7 @@
 		switch(which)
 
 			if("Name")
-				var/stagename = copytext(sanitize(input(targ, "Pick a name","Name") as null|text), 1, MAX_NAME_LEN)
+				var/stagename = copytext(sanitize(input(targ, "Pick a name","Name") as null|text), 1, 52)	//52 characters
```

Comment:
The idea is that you use a define instead of a magic number
putting a comment behind the magic number that explains that it's a magic number doesn't not make it a magic number
use `MAX_NAME_LEN * 2` or even better, go to 
https://github.com/vgstation-coders/vgstation13/blob/c547578f028daf6f75b556474893c816b8649a4b/__DEFINES/setup.dm#L1706-L1707
and add 
```
#define MAX_SPECIAL_NAME_LEN MAX_NAME_LEN * 2
```
and use that instead